### ISSUE TITLE
visio drwaings are empty

### ISSUE THREAD

**Issue by kuchi007** [Tony-SouthFLVMUG/vDiagram2.0#21]
Hi, first I want to thank you for yor effort, real great stuff. 
I have almost empty visio. 
I am using Visio 2016 , Vcenter 7. OS is german.
any idea?
attached I show you the output
[Diagram - 2021_07_29-21_33.zip](https://github.com/Tony-SouthFLVMUG/vDiagram2.0/files/6903344/Diagram.-.2021_07_29-21_33.zip)

**Comment by Tony-SouthFLVMUG** [comment]
Based on what I can see in the drawing it did draw but is missing quite a bit. It might be an English to German conversion thing as all of the code is written in English and it is calling back to the Visio API in English via PowerShell to do the draw. I'm not sure if there is a way for you to add English as a language or even try changing your default language to English. Let me know if this is something you can try. If not let me know I can have you email us the CSV's and we can run a draw for you.

**Comment by kuchi007** [comment]
csvs are captured proper, i have changed now everzthing to english, same result

**Comment by Tony-SouthFLVMUG** [comment]
So you are still getting images but no names below images or missing items still? When drawing, are you using the exact same name in the vCenter field as you did when you captured it? (Example: If you used FQDN to capture then you must use FQDN to draw.)

**Comment by kuchi007** [comment]
yes, i can not change it, but i think now i know why. I have captured everything again with english os settings and skipped sections like datastore cluster info. for example datastore cluster info and export drs rule info gave me as result when capturing not completed. I have drs with rules in place . but the rest seems to be fine.
everything shows now correct, but only if i skip those 2 sections, which I mentioned above. Thanks a lot! Great work
